tools/go-cli/go-whisk-cli/commands/util.go

```diff
@@ -438,9 +438,14 @@ func logoText() string {
     return logo
 }
 
-func printJSON(v interface{}) {
+func printJSON(v interface{}, stream ...io.Writer) {
```

Comment:
@mdeuser, I remember not using the prettyjson function as it would malform user input.